x-pack/plugins/reporting/public/panel_actions/get_csv_panel_action.tsx

```diff
@@ -5,6 +5,7 @@
  */
 import dateMath from '@elastic/datemath';
 import { i18n } from '@kbn/i18n';
+import _ from 'lodash';
```

Comment:
Are we using lodash here without it being required?

Reply:
Wow! We totally were... Thanks for fixing that.

Reply:
We were and I think it is related with https://github.com/lodash/lodash/issues/3052
I've confirmed that lodash is leaking globally. I'll open an issue to fix that after we merge that PR 😃